Binary files term_ref_autocomplete-7.x-1.0-alpha/.DS_Store and term_ref_autocomplete-7.x-1.x-dev/.DS_Store differ
Only in term_ref_autocomplete-7.x-1.x-dev: term.patch
diff -ur term_ref_autocomplete-7.x-1.0-alpha/term_ref_autocomplete.info term_ref_autocomplete-7.x-1.x-dev/term_ref_autocomplete.info
--- term_ref_autocomplete-7.x-1.0-alpha/term_ref_autocomplete.info	2012-04-26 16:16:24.000000000 -0400
+++ term_ref_autocomplete-7.x-1.x-dev/term_ref_autocomplete.info	2013-10-18 21:39:03.000000000 -0400
@@ -7,9 +7,9 @@
 
 dependencies[] = taxonomy
 dependencies[] = field
-; Information added by drupal.org packaging script on 2012-04-26
-version = "7.x-1.0-alpha2"
+; Information added by drupal.org packaging script on 2013-10-19
+version = "7.x-1.0-alpha2+1-dev"
 core = "7.x"
 project = "term_ref_autocomplete"
-datestamp = "1335471384"
+datestamp = "1382146743"
 
diff -ur term_ref_autocomplete-7.x-1.0-alpha/term_ref_autocomplete.module term_ref_autocomplete-7.x-1.x-dev/term_ref_autocomplete.module
--- term_ref_autocomplete-7.x-1.0-alpha/term_ref_autocomplete.module	2012-04-26 16:10:23.000000000 -0400
+++ term_ref_autocomplete-7.x-1.x-dev/term_ref_autocomplete.module	2013-10-02 16:58:20.000000000 -0400
@@ -126,7 +126,6 @@
       'field types' => array('taxonomy_term_reference'),
       'settings' => array(
         'size' => 60,
-        'autocomplete_path' => 'term_ref_autocomplete/autocomplete',
       ),
       'behaviors' => array(
         'multiple values' => FIELD_BEHAVIOR_CUSTOM,
@@ -166,7 +165,7 @@
   $element['names'] = $el_base + array(
     '#type' => 'term_ref_autocomplete_field',
     '#default_value' => implode(', ', $tags),
-    '#autocomplete_path' => $instance['widget']['settings']['autocomplete_path'] . '/' . $field['field_name'],
+    '#autocomplete_path' => 'term_ref_autocomplete/autocomplete/' . $field['field_name'],
     '#size' => $instance['widget']['settings']['size'],
     '#maxlength' => 1024,
     '#element_validate' => array('term_ref_autocomplete_taxonomy_autocomplete_validate'),
